packages/utils/index.js

```diff
@@ -16,13 +18,17 @@ export class FabricWebComponent extends HTMLElement {
             <link
                 rel="stylesheet"
                 type="text/css"
-                href="https://assets.finn.no/pkg/@fabric-ds/css/v0/fabric.min.css"
+                href="https://assets.finn.no/pkg/@fabric-ds/css/v1/fabric.min.css"
             />
         `;
 
-    this.attachShadow({ mode: 'open' }).appendChild(
-      fabricStylesTemplate.content,
-    );
+    this.attachShadow({ mode: 'open' }).appendChild(fabricStylesTemplate.content);
+  }
+}
+
+export class FabricElement extends LitElement {
+  get _fabricStylesheet() {
+    return html`<link href="https://assets.finn.no/pkg/@fabric-ds/css/v1/fabric.min.css" rel="stylesheet" type="text/css">`;
   }
 }
```

Comment:
Base class so that the Fabric URL is only included in a single location and not in every component.